src/config/dynamoInit.js

```diff
@@ -0,0 +1,12 @@
+import AWS from 'aws-sdk';
+
+export const initialDynamo = () => {
+    const { AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, PORT } = process.env;
```

Comment:
this probably fits better in a config file rather than getting it directly from `process.env`.
i would utilise `src/config/...`